recipes/flask-admin/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{%set name = "Flask-Admin" %}
+{%set version = "1.4.2" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "7d1cfdcb29a7135d4275dc22628c0f068cccfdb84dadad885bde685d0511597c" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - flask >=0.7
+    - wtforms
+
+  run:
+    - python
+    - flask >=0.7
+    - wtforms
+    # Required for flask_admin.contrib.appengine:
+    # - wtforms-appengine
+    # Required for flask_admin.contrib.geoa:
+    # - geoalchemy2
+    # - shapely
+    # Required for flask_admin.contrib.mongoengine:
+    # - flask-mongoengine
+    # Required for flask_admin.contrib.peewee and flask_admin.contrib.peeweemodel:
+    # - peewee
+    # - wtf-peewee
+    #  Required for flask_admin.contrib.pymongo:
+    # - pymongo
+    # Required for flask_admin.contrib.sqla and flask_admin.contrib.sqlamodel:
+    # - sqlalchemy
```

Comment:
I'm ok with the way it is; this is a pretty diverse set of dependencies that fit a variety of use cases; there are plenty of folks who might want to use `Flask-Admin` without using Google AppEngine or MongoDB.